### PR TITLE
St martin valba lesson 5 level 1 2

### PR DESCRIPTION
**Pull request by pecoyn**

### CODE REVIEW

File: src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task12.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        int[] number = new int[3];
+
+        int firstRandom = randomNumber;
+        int secondRandom = randomNumber;
+        int thirdRandom = randomNumber;
+
+        number[0] = firstRandom;
+        number[1] = secondRandom;
+        number[2] = thirdRandom;
```
Comment (antdolg1) [inline src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task12.java:19]: In this case array will be filled with three equal numbers. If you want to fill it with different numbers, you need to call `random.nextInt()` every time. E.g.

```
      int firstRandom = random.nextInt();
      int secondRandom = random.nextInt();
      int thirdRandom = random.nextInt();
```

File: src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task13.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import java.util.Random;
+
+public class Task13 {
+    public static void main(String[] args) {
+
+        Random random = new Random();
+        int randomNumber = random.nextInt();
+
+        int[] randoms = {randomNumber, randomNumber, randomNumber};
```
Comment (antdolg1) [inline src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task13.java:11]: Same issue as in task 12.

File: src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task14.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+import java.util.Random;
+
+public class Task14 {
+    public static void main(String[] args) {
+
+        Random random = new Random();
+        int randomNumber = random.nextInt();
+
+        int[] randoms = {randomNumber, randomNumber, randomNumber};
```
Comment (antdolg1) [inline src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task14.java:11]: Same issue as in task 12.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Please check my comments regarding generating random number and apply fixed. Other than that - good job!

File: src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task12.java (hunk 1 above)
Comment (pecoyn) [reply src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task12.java:19]: Done

File: src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task13.java (hunk 2 above)
Comment (pecoyn) [reply src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task13.java:11]: Done

File: src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task14.java (hunk 3 above)
Comment (pecoyn) [reply src/main/java/students/student_Martin_Valba/lesson_5/homework/level_2/Task14.java:11]: Done

**Review by antdolg1 — APPROVED** [review]

**Pull request merged**